docs/content/getting-started/installation/troubleshooting-installation.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Troubleshooting Installation | Dagster
+description: Resources for getting help and troubleshooting common Dagster installation problems, ex. Python and its dependencies
+---
+
+# Troubleshooting Installation
+
+This page covers common troubleshooting issues when installing Dagster. You will also you find resources for getting more help.
+
+---
+
+## Resources
+
+If you have questions about Dagster that are not covered by the documentation, feel welcome to reach out to the team and community by:
+
+- joining the [Dagster Slack community](https://dagster.slack.com/join/shared_invite/zt-1mkwtli11-w4SJ9ucNAyAfl\~Q8qXWz8g#/shared-invite/email) and asking for help on the `#dagster-support` channel.
+- raising a [GitHub issue](https://github.com/dagster-io/dagster/issues)
+
+The team working on Dagster is committed to responding to your questions within a couple of hours during US business days.
+
+---
+
+## For people using Apple M1 computers
+
+If you get an error that reads similar to:
+
+    mach-o, but wrong architecture
```

Comment:
🔧 Can you put this in a code block and specify a language (probably `shell` or `bash`, I don't think it matters much)